assets/js/worker.js

```diff
@@ -1,18 +1,38 @@
-onmessage = function(event) {
-	self.importScripts('https://unpkg.com/lunr/lunr.js');
-	
+let scriptLoaded = false;
+
+onmessage = async function(event) {
+	const scriptUrl = 'https://unpkg.com/lunr/lunr.js';
+  const integrityHash = 'sha256-lDFybwXA6uKm5U3Bl3CUIoafJcrUTyQw0vt92ugMxxc=';
+	if (scriptLoaded) return
```

Comment:
Does this produce the same behaviour as previously? In the previous code, the `lunr()` function would be called for every message event, but in this new version, `lunr()` can only be called once for the lifetime of the worker. Does it actually work the same?